extend-account-console/pom.xml

```diff
@@ -36,19 +36,6 @@
     </dependencies>
 
     <build>
-        <finalName>${project.artifactId}</finalName>
-        
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <excludes>
-                    <exclude>**/keycloak-man/account/resources/content/keycloak-man/KeycloakManLovesJsx.js</exclude>
-                    <exclude>**/keycloak-man/account/resources/content/keycloak-man/KeycloakManLovesJsx.js.map</exclude>
-                    <exclude>**/keycloak-man/account/src/node_modules/**</exclude>
-                    <exclude>**/keycloak-man/account/src/web_modules/**</exclude>
-                    <exclude>**/keycloak-man/account/resources/public/app.css</exclude>
-                </excludes>
-            </resource>
-        </resources>
+        <finalName>keycloak-man-theme</finalName>
```

Comment:
@vmuzikar is this a typo? keycloak-*man*-theme

Reply:
> @vmuzikar is this a typo? keycloak-_man_-theme

Not a typo.  This is the theme for "Keycloak Man", a retired super hero who fought for truth, justice, and the Keycloak Way.